Maquetado/scss/detail.scss

```
@@ -32,11 +38,16 @@
 }
 .bookImage {
   width: 250px;
-  height: 360px;
-  margin-right: 25px;
-  margin-left: 20px;
+  height: 385px;
+  padding: 20px 20px 20px 20px
```

Remember that some css attributes admit "[shorthands](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties)". Padding is one of them.

```suggestion
  padding: 20px;
```